view.c

```diff
@@ -138,13 +138,19 @@ view_is_transient_for(struct cg_view *child, struct cg_view *parent) {
 void
 view_damage_surface(struct cg_view *view)
 {
-	output_damage_view_surface(view->server->output, view);
+	struct cg_output *output;
+	wl_list_for_each(output, &view->server->outputs, link) {
+		output_damage_view_surface(output, view);
+	}
 }
 
 void
 view_damage_whole(struct cg_view *view)
 {
-	output_damage_view_whole(view->server->output, view);
+	struct cg_output *output;
+	wl_list_for_each(output, &view->server->outputs, link) {
+		output_damage_view_whole(output, view);
```

Comment:
Can we find out what output the damage occurs on and only call `output_damage_view_surface` on that output?

Reply:
Answered in other comment.